Frontend/CO.CDP.OrganisationApp/Pages/Forms/FormElementDateInputModel.cs

```diff
@@ -42,11 +50,34 @@ public override void SetAnswer(FormAnswer? answer)
             Month = answer.DateValue.Value.Month.ToString();
             Year = answer.DateValue.Value.Year.ToString();
         }
+        else if (RedirectFromCheckYourAnswerPage && IsRequired == false)
+        {
+            HasValue = false;
+        }
     }
 
     public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
     {
-        if (CurrentFormQuestionType == FormQuestionType.Date && IsRequired == true && string.IsNullOrWhiteSpace(DateString))
+        if (CurrentFormQuestionType != FormQuestionType.Date)
```

Comment:
Why in the DateInputModel would we reach the situation where CurrentFormQuestionType is not equal to Date?

Reply:
Good point, it may be obsolete now, we had a situation where all forms partial were executing validations.
I will check in my next changes and do clean up